kong/db/strategies/off/init.lua

```diff
@@ -45,6 +45,21 @@ local _mt = {}
 _mt.__index = _mt
 
 
+local DEFAULT_WORKSPACE_ID = "00000000-0000-0000-0000-000000000000"
+
+
+local function get_default_workspace()
+  local ws_id = kong.default_workspace
+
+  if ws_id == DEFAULT_WORKSPACE_ID then
+    local res = assert(kong.db.workspaces:select_by_name("default"))
+    ws_id = res.id
```

Comment:
`kong.default_workspace = ws_id`?

Reply:
changed to return a table, see: https://github.com/Kong/kong/pull/13157/commits/fe0cc9b901e8de9bf7c2b8cf537b6fe43022a26a